packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/repository.ts

```diff
@@ -127,6 +141,7 @@ export interface SavedObjectsRepositoryOptions {
 export const DEFAULT_REFRESH_SETTING = 'wait_for';
 export const DEFAULT_RETRY_COUNT = 3;
 
+const MAX_CONCURRENT_ALIAS_DELETIONS = 10;
```

Comment:
This is the same limit on the number of concurrent alias deletions as used for [updating objects' spaces](https://github.com/elastic/kibana/blob/main/packages/core/saved-objects/core-saved-objects-api-server-internal/src/lib/update_objects_spaces.ts#L71). We could export the constant from here and reuse that as a default everywhere we implement a `pMap`.